cPP_DSC.adoc

```diff
@@ -589,22 +589,18 @@ The conventions used in descriptions of the SFRs are as follows:
 * Unaltered SFRs are stated in the form used in [CC2] or their extended component definition (ECD);
 * Refinement made in the PP: the added/removed text is indicated with *bold text*/[.line-through]#strikethroughs#. When text is substituted (i.e. some text is added in place of some other text, which is then deleted), only the added text is included;
 
-Note that a refinement is also used to indicate cases where the PP replaces an assignment defined for an SFR in [CC2] and replaces it with a selection;
```

Comment:
This sentence was removed because it's explained better below ("If the original SFR defined an assignment operation which was completed by the PP author by redefining it as a selection operation ...")